tests/serialize/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
About the package.json, that's usually done when you wanna include scripts from 3rd party libraries in your module. The "children" scripts inherit the parent environment with all the libraries and globals, it's pretty much as if you were running them in the main folder. I was just trying to not pollute the main json with too much stuff, but if you prefer I can move everything.